ckanext/geocat/utils/harvest_helper.py

```diff
@@ -50,16 +50,22 @@ def create_activity(package_id, message):
 
 
 def check_package_change(existing_pkg, dataset_dict):
-    # Ensure to clear the key values if they are not
-    # in the new dataset_dict
-    for key in existing_pkg.keys():
-        if key not in dataset_dict or not dataset_dict[key]:
-            existing_pkg[key] = ''
+    # Check if the URL has changed
+    existing_pkg_url = existing_pkg.get('url', '')
+    dataset_dict_url = dataset_dict.get('url', '')
+    if existing_pkg_url != dataset_dict_url:
+        msg = "dataset url value changed from '{}' to '{}'".format(
+            existing_pkg_url or "", dataset_dict_url or "")
```

Comment:
`format()` can handle non-string values, so I think it's better to just put this here:
 
```suggestion
            existing_pkg_url, dataset_dict_url)
```

This way, if the value changes to `None`, we will log that, and not `''`, which could be misleading.

Reply:
Could we add this update to `check_package_change` in ckanext-dcatapchharvest as well? It would be good to keep them aligned with one another.

Reply:
yes, we need to update dcat harvester as well, I will at it there in new ticket